apps/api/src/app/events/usecases/message-matcher/message-matcher.usecase.spec.ts

```diff
@@ -12,7 +12,14 @@ describe('Message filter matcher', function () {
   const createExecutionDetails = {
     execute: sinon.stub(),
   };
-  let messageMatcher = new MessageMatcher(undefined as any, createExecutionDetails as any, undefined as any);
+  let messageMatcher = new MessageMatcher(
+    undefined as any,
+    createExecutionDetails as any,
+    undefined as any,
+    undefined as any,
+    undefined as any,
+    undefined as any
+  );
```

Comment:
In a tech debt task we could rewrite this test suite to avoid to use `any` and also make it more flexible the injection of the stubs for the different tests.